modules/core/include/opencv2/core/ocl.hpp

```diff
@@ -277,6 +279,8 @@ class CV_EXPORTS Context
     /** @returns cl_context value */
     void* ptr() const;
 
+    /** @returns Property specified on context creation */
+    void* getProperty(long propertyId) const;
```

Comment:
As we want to work with OpenCL enums lets use more specific method name `getOpenCLContextProperty()` and add note to doc string about returned values and passed parameter (extract properties through `clGetContextInfo(CL_CONTEXT_PROPERTIES)` call)